### PR TITLE
Release Notes of 2.1.0

### PR DESCRIPTION
**Pull request by pascal-brand-st-dev**
Signed-off-by: Pascal Brand <pascal.brand@st.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,67 @@
-# OP-TEE - version 2.0.0
+# OP-TEE - version 2.1.0
+
+## New features
+
+* New supported platform: Xilinx Zynq UltraScale+ MPSOC
+
+* GCC5 support
+
+* Non Linear Mapping support: In TA user-mode, the physical to virtual
```
Comment (jenswi-linaro) [inline CHANGELOG.md]: s/TA user-mode/OP-TEE kernel mode/

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,67 @@
...
+| ---- | ------- |
+| Standard tests | The [optee_test][optee_test] project. |
+| Extended tests | optee_test with tests from the GlobalPlatform™ TEE Initial Configuration Test Suite v1.1.0.4. |
+
+*	ARM Juno Board (vexpress-juno), standard.
+*	Foundation Models (vexpress-fvp), standard tests + extended tests,
+	using FVP ARM V8 Foundation Platformr0p0 (platform build 10.0.37)
+*	FSL ls1021a (ls-ls1021atwr), standard tests.
+*	HiKey (hikey), standard + extended tests.
+*	MTK8173-EVB (mediatek-mt8173), not tested.
```
Comment (jbech-linaro) [inline CHANGELOG.md]: Should we put the non-tested below the rest. If you quickly look at the section header name and glance through this, you might miss the "not tested" part.

**Comment by jbech-linaro** [comment]
Just one minor comment, otherwise I think you've captured it all.

`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

**Comment by d3zd3z** [comment]
Do we want to include the AOSP and/or oe-optee in these release notes?

**Pull request merged**